products/BellHybrid/apps/application-bell-settings/models/FrontlightModel.cpp

```diff
@@ -56,17 +56,17 @@ namespace app::bell_settings
     }
     void FrontlightModel::setStatus(bool onOff)
     {
-        app->bus.sendUnicast(std::make_shared<sevm::ScreenLightSettingsControlMessage>(
+        app->bus.sendUnicast(std::make_shared<sevm::ScreenLightControlMessage>(
                                  onOff ? screen_light_control::Action::turnOn : screen_light_control::Action::turnOff),
                              service::name::evt_manager);
     }
     void FrontlightModel::setMode(screen_light_control::ScreenLightMode mode)
     {
-        app->bus.sendUnicast(std::make_shared<sevm::ScreenLightSettingsControlMessage>(
-                                 mode == screen_light_control::ScreenLightMode::Automatic
-                                     ? screen_light_control::Action::enableAutomaticMode
-                                     : screen_light_control::Action::disableAutomaticMode),
-                             service::name::evt_manager);
+        app->bus.sendUnicast(
+            std::make_shared<sevm::ScreenLightControlMessage>(mode == screen_light_control::ScreenLightMode::Automatic
+                                                                  ? screen_light_control::Action::enableAutomaticMode
+                                                                  : screen_light_control::Action::disableAutomaticMode),
+            service::name::evt_manager);
```

Comment:
Clang format's job? ;)
What do you think about calculating `const auto action` above to prevent this to happen?